axelor-base/src/main/resources/demo/import_Partners/base_partner.csv

```diff
@@ -0,0 +1,220 @@
+importId;partnerTypeSelect;isContact;isCustomer;isSupplier;isEmployee;isFactor;name;firstName;titleSelect;languageCode;address;email;fixedPhone;mobilePhone;invoiceSendingFormatSelect;inPaymentMode.importId;outPaymentMode.importId;paymentCondition.importId;partnerCategory.code;currency.code;blocking.importId;rejectCounter;companySet_importId;customerAccount_code;supplierAccount_code;mainPartner_importId;customerTypeSelect;supplierTypeSelect;industrySector.importId;user.importId;team.importId;source.id;fiscalPosition.importId;picture_fileName;invoicesCopySelect;factorizedCustomer;companyID
```

Comment:
I think there is an issue with all your file's path: 
every file should be located at /ressources/demo/fr/
And you should not have any folder into this folder, only files. 
-> Ideally we should not touch the base xml configs